conversions/Backwards/src/main/java/com/agentdid127/resourcepack/backwards/impl/ModelConverter.java

```diff
@@ -54,233 +57,254 @@ protected void findFiles(Path path) throws IOException {
+                            content = content.replaceAll("item/", "items/");
+                            content = content.replaceAll("block/", "blocks/");
+                            content = content.replaceAll(" ", "_");
+                        }
+                        // handle the remapping of textures, for models that use default texture names
+                        jsonObject = packConverter.getGson().fromJson(content, JsonObject.class);
+                        if (jsonObject.has("textures") && jsonObject.get("textures").isJsonObject()) {
+                            NameConverter nameConverter = packConverter.getConverter(NameConverter.class);
 
+                            JsonObject initialTextureObject = jsonObject.getAsJsonObject("textures");
+                            JsonObject textureObject = initialTextureObject.deepCopy();
+                            for (Map.Entry<String, JsonElement> entry : initialTextureObject.entrySet()) {
+                                String value = entry.getValue().getAsString();
+                                PackConverter.log(entry.getKey() + ": " + entry.getValue());
+                                textureObject.remove(entry.getKey());
+                                // 1.13 Mappings
+                                if (version < Util.getVersionProtocol(packConverter.getGson(), "1.13")) {
+                                    if (value.startsWith("block/")) {
+                                        value = "block/" + nameConverter.getBlockMapping()
+                                                .remap(value.substring("block/".length())).toLowerCase()
+                                                .replaceAll("[()]", "");
+                                        PackConverter.log(value.substring("block/".length()).toLowerCase()
+                                                .replaceAll("[()]", ""));
+                                        PackConverter.log(nameConverter.getBlockMapping()
+                                                .remap(value.substring("block/".length())).toLowerCase()
+                                                .replaceAll("[()]", ""));
+                                    } else if (value.startsWith("item/")) {
+                                        value = "item/" + nameConverter.getItemMapping()
+                                                .remap(value.substring("item/".length())).toLowerCase()
+                                                .replaceAll("[()]", "");
+                                    } else
+                                        value = value.toLowerCase().replaceAll("[()]", "");
+                                }
 
-                    String content = jsonObject.toString();
-                    if (version < Util.getVersionProtocol(packConverter.getGson(), "1.13")) {
-                        content = content.replaceAll("item/", "items/");
-                        content = content.replaceAll("block/", "blocks/");
```

Comment:
Folder changes are required for versions less than 1.13

Reply:
I didn't delete it, check again